packages/flutter/lib/src/material/slider.dart

```diff
@@ -475,13 +475,22 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
   Timer? interactionTimer;
 
   final GlobalKey _renderObjectKey = GlobalKey();
+
   // Keyboard mapping for a focused slider.
-  final Map<ShortcutActivator, Intent> _shortcutMap = const <ShortcutActivator, Intent>{
+  final Map<ShortcutActivator, Intent> _traditionalNavShortcutMap = const <ShortcutActivator, Intent>{
```

Comment:
This can be a static const since it is const anyway?

Reply:
done